lib/addon-modes/browser.js

```diff
@@ -1,10 +1,11 @@
 var filterInitializers = require('../broccoli/filter-initializers');
+
 module.exports = {
   /**
    * Filters out initializers and instance initializers that should only run in
    * FastBoot mode.
    */
-  treeForApp: function(tree) {
-    return filterInitializers(tree, 'server');
+  preconcatTree: function(tree) {
```

Comment:
Why are we creating our own hook here? In general, I would prefer to stick with the normal hooks like `treeForApp`.  If we need to make extra hooks upstream because something is not possible otherwise let's do it, but I'm not seeing exactly why we would want to do this before `concatFiles` vs `treeForApp`...